frontend/src/toolbar/elements/heatmapToolbarMenuLogic.ts

```diff
@@ -514,6 +515,11 @@ export const heatmapToolbarMenuLogic = kea<heatmapToolbarMenuLogicType>([
             }
         },
 
+        refreshClickmap: () => {
```

Comment:
oh, no it's new :rolling_on_the_floor_laughing:  
  
in which case i'd check if click map is enabled in here and return early if it isn't  
then the caller doesn't need to care if it should call